concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetBakerListTest.java

```diff
@@ -0,0 +1,67 @@
+package com.concordium.sdk;
+
+import com.concordium.grpc.v2.*;
+import com.concordium.sdk.exceptions.BlockNotFoundException;
+import com.concordium.sdk.requests.BlockHashInput;
+import com.concordium.sdk.responses.BakerId;
+import com.google.common.collect.ImmutableList;
+import io.grpc.ManagedChannel;
+import io.grpc.inprocess.InProcessChannelBuilder;
+import io.grpc.inprocess.InProcessServerBuilder;
+import io.grpc.stub.StreamObserver;
+import io.grpc.testing.GrpcCleanupRule;
+import lombok.var;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.AdditionalAnswers.delegatesTo;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ClientV2GetBakerListTest {
```

Comment:
Please add one sentence of what the intention is with the test. I.e. testing the mapping functionality.

Reply:
done